modules/persistent-storage-csi-gcp-hyperdisk-limitations.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-gcp-pd.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="persistent-storage-csi-gcp-hyperdisk-limitations_{context}"]
+= C3 and N4 instance type limitations
+The GCP PD CSI driver support for the C3 instance type for bare metal and N4 machine series have the following limitations:
+
+* Cloning volumes is not supported when using storage pools.
+
+* For cloning or resizing, hyperdisk-balanced disks original volume size must be 6Gi or greater.
+
+* A GCP cluster with n4-standard-2 or n4-standard-4 nodes can erroneously exceed the maximum attachable disk number, which should be 16.
+
```

Comment:
c3 also has the attach disk number issue. https://issues.redhat.com/browse/OCPBUGS-39258

Reply:
Fixed.